build.psake.ps1

```diff
@@ -256,13 +256,40 @@ Task Test -depends Analyze -requiredVariables TestRootDir, ModuleName {
         Microsoft.PowerShell.Management\Push-Location -LiteralPath $TestRootDir
 
         if ($TestOutputFile) {
-            $TestResult = Invoke-Pester -OutputFile $TestOutputFile -OutputFormat $TestOutputFormat -PassThru -Verbose:$VerbosePreference
+            $Testing = @{
+                OutputFile   = $TestOutputFile
+                OutputFormat = $TestOutputFormat
+                PassThru     = $true
+                Verbose      = $VerbosePreference
+            }
         }
         else {
-            $TestResult = Invoke-Pester -PassThru -Verbose:$VerbosePreference
+            $Testing = @{
+                PassThru     = $true
+                Verbose      = $VerbosePreference
+            }
         }
 
-        Assert ($TestResult.FailedCount -eq 0) "One or more Pester tests failed, build cannot continue."
+        if ($CodeCoverageStop -or ($null -eq $CodeCoverageStop)) {
```

Comment:
That looks weird.  :-)  I get it but still might be worth a comment about checking for $true/$false/$null.